### DISCUSSION TITLE
Add option to move the top bar to the bottom

### DISCUSSION THREAD

**Discussion by zipaio** [jocmp/capyreader#599] [Feature Requests]
First of all, great work! It would be nice to have an option to move the top bar to the bottom area, as it would be more comfortable to use the app with one hand. For example, being able to press "mark all as read" without needing to use the other hand or stretch the finger.

Thank you!

**Comment by issteve** [comment]
@jocmp @zipaio  Maybe it would be easier to implement and as a first step okay to have only the "mark all as read" button as a second single floating button at the bottom? 
(Example from SparseRSS having it as only a curved arrow just above the navigation buttons, but it could be a real button too of course):


![Screenshot_20250130-181408](https://github.com/user-attachments/assets/3d69fde1-ff9e-4698-a18d-b6e3bf5b1b8b)

**Reply by zipaio** [reply]
A floating button for 'mark all as read' at the bottom sounds like a solid first step. While that would improve one-handed use, having the top bar’s other functions (like the feeds menu and search) accessible from the bottom in the future would make navigation even more intuitive. Thanks for considering this!

**Comment by issteve** [comment]
It seems it's already on the list: https://github.com/jocmp/capyreader/issues/795

**Reply by issteve** [reply]
@jocmp excuse me to ping this wish again, but as the article bar could be moved to the bottom now, could there be any chance in the future to have the top bar in the feed view at the bottom, too? 
Or maybe at least the "mark all as read" button? Thank you for your consideration!

**Reply by jocmp** [reply]
Hey @issteve - I don't have any plans to move the top bar to the bottom since it would require a lot of rework and more settings. It's not prioritized, but I do still like the idea of the "mark all as read" as a floating action button. That's tracked here:

- https://github.com/jocmp/capyreader/issues/891

**Reply by issteve** [reply]
Thank you very much for checking back, your answer, for keeping the option of the "mark all as read" as a floating action button and giving me the tracking (I should have found by myself🫣)! I'm patiently looking forward to find it "on deck" one day again... 😘
